### PR TITLE
Coding style

### PR DESCRIPTION
**Pull request by PierreRambaud**
Improve code examples (PSR2) and fix markdown error in README.

### CODE REVIEW

File: src/content/1.7/modules/creation/displaying_content_in_front_office.md [hunk 1]
```diff
@@ -260,26 +268,33 @@ which is a must when using the translation tool.
...
 
-    A front-end controller must be a class that extends the ModuleFrontController class.
-    That controller must have one method: `initContent()`, which calls the parent class' `initContent()` method...
-    ...which then calls the `setTemplate()` method with our `display.tpl` file.
+
+```
+A front-end controller must be a class that extends the ModuleFrontController class.
+That controller must have one method: `initContent()`, which calls the parent class' `initContent()` method...
+...which then calls the `setTemplate()` method with our `display.tpl` file.
+```
```
Comment (Quetzacoalt91) [inline src/content/1.7/modules/creation/displaying_content_in_front_office.md]: I'm not sure this part should be in a code div

File: src/content/1.7/modules/creation/displaying_content_in_front_office.md [hunk 2]
```diff
@@ -260,26 +268,33 @@ which is a must when using the translation tool.
...
     }
+}
+```
 
-    display.tpl
-    Welcome to my shop!
+- **display.tpl*
+```
+Welcome to my shop!
+```
```
Comment (Quetzacoalt91) [inline src/content/1.7/modules/creation/displaying_content_in_front_office.md]: One carriage return is missing, we get unexpected results

capture du 2018-04-26 17-42-08

**Comment by Quetzacoalt91** [comment]
Thank you @PierreRambaud

File: src/content/1.7/modules/creation/displaying_content_in_front_office.md (hunk 2 above)
Comment (PierreRambaud) [reply src/content/1.7/modules/creation/displaying_content_in_front_office.md]: Look weird, didn't have this result on my hugo instance :/

File: src/content/1.7/modules/creation/displaying_content_in_front_office.md (hunk 1 above)
Comment (PierreRambaud) [reply src/content/1.7/modules/creation/displaying_content_in_front_office.md]: Not sure too, but just formatting already existed.

**Pull request merged**